hoot-core/src/main/cpp/hoot/core/scoring/BaseComparator.cpp

```diff
@@ -68,17 +68,17 @@ void BaseComparator::_calculateColor(double v, double max, QRgb& c) const
     }
     if (v < max / 3.0)
     {
-      int r = v / (max / 3.0) * 255.0;
+      double r = v / (max / 3.0) * 255.0;
```

Comment:
`qRgb()` takes `int`s as parameters.  The whole result needs to be casted to an integer like other examples above.